openpathsampling/numerics/wham.py

```diff
@@ -155,18 +155,18 @@ def prep_reverse_cumulative(self, df, cutoff=None, tol=None):
             )
         else:
             # clear duplicates of leading values
-            test_f = lambda val1, val2, val_max : (
+            test_f = lambda val1, val2, val_max: (
                 abs(val1 - val2) > tol or abs(val1 - val_max) > tol
             )
             cleaned_df = cleaned_df.apply(
-                lambda s : [
-                    s.iloc[i] if test_f(s.iloc[i], s.iloc[i+1], s.max()) else 0.0
+                lambda s: [
+                    s.iloc[i] if test_f(s.iloc[i], s.iloc[i+1], s.max())
+                    else 0.0
```

Comment:
If you're going to split this line for PEP8, please start the extra line with the `if` and keep the `else` on the same line with it